common/history/population/0_dvg_indonesia.txt

```diff
@@ -116,4 +136,19 @@
 		effect_starting_pop_wealth_low = yes
 		effect_starting_pop_literacy_baseline = yes
 	}
+
+	c:BEI = { #Burgundian East India Company
+		effect_starting_pop_wealth_high = yes
```

Comment:
What is the reason for giving high-wealth pops? This is only for high-wealth countries.